src/test/resources/results/scriptCommand/testScriptCommandForSchemaEvolution_table_deleted/Makefile

```diff
@@ -0,0 +1,7 @@
+
+
+
+
+
+            evolve-schema-all:
+    
```

Comment:
I think we saw an error in MDT where we had no target like this script here. I think if there is "no change" we should just do nothing as opposed to fail with an error. But we can take this up later

Reply:
In case there is no schema changes meaning no changes in any tables then we get `evolve-schema-all:`. This won't give any error. Once this is executed with make command it will give below message 
`make: Nothing to be done for `evolve-schema-all'.`
This is added as part of https://phdata.atlassian.net/browse/STREAM-60

Reply:
> make: Nothing to be done for evolve-schema-all'.`

Gotcha, I was thinking this exited non-zero for some reason.... I see that is not the case anymore.